fault/tester.py

```diff
@@ -536,6 +536,26 @@ def internal(self, *args):
         # return a SelectPath containing the desired path
         return SelectPath([self.circuit] + list(args))
 
+    def __call__(self, *args, **kwargs):
+        """
+        Poke the inputs of the circuit using *args (ordered, anonymous input
+        reference, excluding clocks) and **kwargs.  **kwargs will overwrite any
+        inputs written by *args.
+
+        Evaluate the circuit
+
+        Return the "peeked" output(s) of the circuit (tuple for multiple
+        outputs)
+        """
+        for arg, port in zip(args, self._circuit.interface.outputs()):
```

Comment:
I don't see `kwarg` override happening here?

Reply:
oops forgot it! heh